torchtitan/experiments/llama4/model/moe.py

```diff
@@ -17,53 +17,72 @@ def __init__(
         dim: int,
         hidden_dim: int,
         num_experts: int,
+        fused: bool,
     ):
         super().__init__()
         self.num_experts = num_experts
         self.w1 = nn.Parameter(torch.empty(num_experts, dim, hidden_dim))
         self.w2 = nn.Parameter(torch.empty(num_experts, hidden_dim, dim))
         self.w3 = nn.Parameter(torch.empty(num_experts, dim, hidden_dim))
+        self.fused = fused
```

Comment:
same as earlier - self.use_grouped_gemm is a lot clearer. fused is rather nebulous.